main.nf

```diff
@@ -184,57 +187,69 @@ if (params.pseudo_aligner && params.pseudo_aligner != 'salmon') {
     exit 1, "Invalid pseudo aligner option: ${params.pseudo_aligner}. Valid options: 'salmon'"
 }
 
-if (params.star_index && params.aligner == 'star' && !params.skipAlignment) {
+if (params.star_index && params.aligner == 'star' && !params.skip_alignment) {
     if (hasExtension(params.star_index, 'gz')) {
-        star_index_gz = Channel
+        Channel
             .fromPath(params.star_index, checkIfExists: true)
             .ifEmpty { exit 1, "STAR index not found: ${params.star_index}" }
+            .set { star_index_gz }
     } else {
-        star_index = Channel
+        Channel
             .fromPath(params.star_index, checkIfExists: true)
             .ifEmpty { exit 1, "STAR index not found: ${params.star_index}" }
+            .set { star_index }
     }
-} else if (params.hisat2_index && params.aligner == 'hisat2' && !params.skipAlignment) {
+} else if (params.hisat2_index && params.aligner == 'hisat2' && !params.skip_alignment) {
     if (hasExtension(params.hisat2_index, 'gz')) {
-        hs2_indices_gz = Channel
+        Channel
             .fromPath("${params.hisat2_index}", checkIfExists: true)
             .ifEmpty { exit 1, "HISAT2 index not found: ${params.hisat2_index}" }
+            .set { hs2_indices_gz }
     } else {
-        hs2_indices = Channel
+        Channel
             .fromPath("${params.hisat2_index}*", checkIfExists: true)
             .ifEmpty { exit 1, "HISAT2 index not found: ${params.hisat2_index}" }
+            .set { hs2_indices }
     }
 }
-if ( params.fasta && !params.skipAlignment) {
+if (params.fasta && !params.skip_alignment) {
     if (params.additional_fasta) {
-        if ( hasExtension(params.additional_fasta, "gz" )){
-            Channel.fromPath(params.additional_fasta)
+        if (hasExtension(params.additional_fasta, "gz")) {
+            Channel
+                .fromPath(params.additional_fasta)
                 .ifEmpty { exit 1, "Additional Fasta file not found: ${params.additional_fasta}" }
-                .into { additional_fasta_gz }
-            Channel.fromPath(params.fasta, checkIfExists: true)
+                .set { additional_fasta_gz }
+            Channel
+                .fromPath(params.fasta, checkIfExists: true)
                 .ifEmpty { exit 1, "Genome fasta file not found: ${params.fasta}" }
                 .set { genome_fasta_gz }
         } else {
-            Channel.fromPath(params.additional_fasta)
+            Channel
+                .fromPath(params.additional_fasta)
                 .ifEmpty { exit 1, "Additional Fasta file not found: ${params.additional_fasta}" }
-                .into { ch_additional_fasta_for_gtf; ch_additional_fasta_to_concat }
-            Channel.fromPath(params.fasta, checkIfExists: true)
+                .into { ch_additional_fasta_for_gtf
+                        ch_additional_fasta_to_concat }
```

Comment:
```suggestion
                .into { ch_additional_fasta_for_gtf;
                        ch_additional_fasta_to_concat }
```

Reply:
ok, as I see you have removed them in more places, probably it doesn't!

Reply:
Yep. It works without semicolons if you put them on different lines :)